### DISCUSSION TITLE
Hi, i have stuck

### DISCUSSION THREAD

**Discussion by ghost** [soranoo/TradingView-Free-Webhook-Alerts#12] [Q&A]
26-11-2022 20:10:32 | INFO      | Initializing...
26-11-2022 20:10:34 | INFO      | Listening to IMAP server(outlook.office365.com)...

what may cause this problem? 
I receive mails but don't get redirection on webhook
logs are:
26-11-2022 19:56:05 | INFO      | Initializing...
26-11-2022 19:56:07 | INFO      | Listening to IMAP server(outlook.office365.com)...
26-11-2022 20:10:32 | INFO      | Initializing...
26-11-2022 20:10:34 | INFO      | Listening to IMAP server(outlook.office365.com)...

**Comment by Andreymyski** [comment]
Hello. Make sure that the username and password are only in config. Don't touch anything else.

**Reply by ghost** [reply]
> Hello. Make sure that the username and password are only in config. Don't touch anything else.

I recreated config again. This time I didn't write webhook, but it still doesn't link to mail.

**Comment by Andreymyski** [comment]
I do not know what could be.

**Reply by ghost** [reply]
> I do not know what could be.

script doesn't read mail. Is there should be any confirmations that script had logged into account?

**Comment by Andreymyski** [comment]
Follow the instructions

**Reply by ghost** [reply]
> Follow the instructions

![underlined](https://user-images.githubusercontent.com/74045824/204135608-41ed73e7-e38b-459f-9a48-f0a19e84ecee.png)
are those underlines ok? I have installed by pip from requirements

**Comment by Andreymyski** [comment]
It's a little different for me. I figured out what your problem is. Install all the dependencies. here is the command: pip install -r requirements.txt

**Comment by soranoo** [comment]
Please ensure the email address you set in the config file(`email_address`) is the same as your TradingView SMS email.
The SMS email is located in **Profile Setting** -> **PRIVATE DETAILS**

You may try another email service provider such as Google Gmail if the program still doesn't work.

**Reply by soranoo** [reply]
And also you may test your signal or the program using a webhook test service such as [webhook.site](https://webhook.site/) instead of using your production webhook.
